server/src/test/java/org/elasticsearch/action/admin/indices/mapping/put/PutMappingRequestTests.java

```diff
@@ -70,43 +67,13 @@ public void testBuildFromSimplifiedDef() {
         assertEquals("mapping source must be pairs of fieldnames and properties definition.", e.getMessage());
     }
 
-    public void testToXContent() throws IOException {
-        PutMappingRequest request = new PutMappingRequest("foo");
-
-        XContentBuilder mapping = JsonXContent.contentBuilder().startObject();
-        mapping.startObject("properties");
-        mapping.startObject("email");
-        mapping.field("type", "text");
-        mapping.endObject();
-        mapping.endObject();
-        mapping.endObject();
-        request.source(mapping);
+    public void testFromXContent() throws IOException {
```

Comment:
Reading this I was wondering if we need this test at all anymore. The way I understand the server-side request class, is just stores the mapping source definition as a Json String, which is converted back to a map somewhere else I suppose. Let me know what you think, maybe we don't need this here.

Reply:
After looking at the logic more closely, I agree this test doesn't seem too valuable.

Reply:
It might even be worth looking into why we store the source as a json string when we later parse it to a map. But that's digressing from this PR, I can make a note of that and take a look.